pom.xml

```diff
@@ -36,7 +36,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+            <version>3.0.4</version>
```

Comment:
Declaring the version of this dependency is unnecessary, as it is actively managed by spring-boot-starter-parent. In fact, version 3.0.4 here likely conflicts with version 3.0.6 that spring-boot-starter-parent prefers to bring in.

Reference: 
https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#appendix.dependency-versions.coordinates


```suggestion
```

Reply:
Yep. removed.